src/main/java/com/mathworks/ci/actions/RunMatlabBuildAction.java

```diff
@@ -29,6 +29,15 @@ public class RunMatlabBuildAction {
         "+ciplugins/+jenkins/BuildReportPlugin.m";
     private static String TASK_RUN_PROGRESS_PLUGIN = 
         "+ciplugins/+jenkins/TaskRunProgressPlugin.m";
+    private String actionID;
+
+    public String getActionID(){
+        return this.actionID;
+    }
```

Comment:
`actionID` seems like implementation detail here rather than part of the class's interface that we want to expose. I think my preference is to remove the setter and getter and just set the value in the constructor as currently they're adding verbosity without much benefit.